feature/preview/src/test/java/com/google/jetpackcamera/feature/preview/PreviewViewModelTest.kt

```diff
@@ -89,15 +89,15 @@ class PreviewViewModelTest {
     @Test
     fun startVideoRecording() = runTest(StandardTestDispatcher()) {
         previewViewModel.startCameraUntilRunning()
-        previewViewModel.startVideoRecording()
+        previewViewModel.startVideoRecording(null, true) {}
```

Comment:
Can we add a separate test for External Intent Video Capture mode, like we have for image capture?

Reply:
Expanded VideoRecordingDeviceTest